client/webpack.config.js

```diff
@@ -127,7 +134,7 @@ module.exports = {
   devServer: {
     writeToDisk: true
   },
-  devtool: DEV ? 'cheap-module-eval-source-map' : 'source-map'
+  devtool: DEV ? 'cheap-module-source-map' : 'source-map'
```

Comment:
Why? :)

Reply:
`cheap-module-eval-source-map` is not a valid devtool in webpack@5: https://webpack.js.org/configuration/devtool/